public/src/components/dashboard.jsx

```diff
@@ -69,6 +70,7 @@ export default class Dashboard extends Component {
         <div className="dashboard-window">
           <div className="report-tile shadow"><PieReport auth={this.props.getAuth} title="Recent Ingredients:" type="Meal"/></div>
           <div className="report-tile report-tile-wide shadow"><EntryList auth={this.props.getAuth} /></div>
+          <div className="report-tile report-tile-wide shadow"><WaterReport data={this.props.data} auth={this.props.getAuth}/></div>
```

Comment:
where is data coming from?
last we discussed this morning, the data was going to be handled by the report component, isn't it?